DQMOffline/Trigger/src/HLTTauDQMPath.cc

```diff
@@ -54,6 +54,8 @@ namespace {
           continue;
         if (type == "HLTTriggerTypeFilter" || type == "HLTBool")
           continue;
+        if (iLabel->find('-') == 0)  // ignore
```

Comment:
Aren't there modules with a "-" somewhere in the name? Wouldn't it be safer to check that they just don't start with a "-"? 
(perhaps @cms-sw/hlt-l2 could remind me if a "-" in the name is just forbidden for hlt modules, in which case the proposed syntax is ok)

Reply:
Hi @perrotta 

> Wouldn't it be safer to check that they just don't start with a "-"?

I think that's what this check is doing: it checks if character `-` is at position `0` using `find`, and if so it skips the word.

> Aren't there modules with a "-" somewhere in the name?

If you mean a `-` somewhere in the middle of the module name/label, as far as I know that's not possible for any module, not just HLT (that works for `_`, of course, but not for other special characters like `-`).